docs/examples/topic/README.md

```diff
@@ -33,6 +33,9 @@ construct used by higher-level objects, such as `Channel`.
    1. If you are using standard Kubernetes secrets, but want to use a
       non-default one, update `secret` with your own secret.
 
+   1. If you are [managing multiple project](../../install/managing-multiple-projects.md), be careful to specify the `project`
+   where you want your Google Cloud resources resides.
+
```

Comment:
The gcloud command in Verify should include a --project (--topic-project ?) flag.